### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by djo1975**
-In this project we got a chance to optimize some slow queries in our database.

Screenshots:

1AI
analyze1
analyzetwo
vetpopulate
vetpopulatetwo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ After you're finished please remove all the comments and instructions!
 
 # 📖 Vet-clinic-database <a name="about-project"></a>
 
-In this project, I added some many-to-many relationships and write more complex queries to use them to answer questions.
+In this project we got a chance to optimize some slow queries in our database.
```
Comment (afanimartin) [inline README.md:52]: - Add explicit commands for `Getting started`, `Installing` sections. Make it easy for someone getting started with your project.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @djo1975  and @Marcraphael12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Screenshots added to PR description
- Professional readme template used as recommended

## Required Changes ♻️
_Check the comments under the review._
- Please add more screenshots in your PR description for the query results. You are supposed to run three commands.
`SELECT COUNT(*) FROM visits where animal_id = 4;`
`SELECT * FROM visits where vet_id = 2;`
`SELECT * FROM owners where email = 'owner_18327@mail.com';`
For each command. Add screenshot for query result before and after making changes to the code. A total of 6 screenshots are needed. @Marcraphael12 it would be nice if you make the changes. I only see one commit from you.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi TEAM
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge:shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights



✔️ Professional Readme file
✔️ No linters error


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Comment by djo1975** [comment]
THX :)

**Pull request merged**